test/quantization/pt2e/test_quantize_pt2e.py

```diff
@@ -1750,3 +1754,54 @@ def test_qat_resnet18(self):
             self._verify_symmetric_qnnpack_qat_numerics(
                 m, example_inputs, is_per_channel=True,
             )
+
+@skipIfNoDynamoSupport
```

Comment:
Lets start splitting tests. I would suggest `test_x86inductor_quantizer.py`

Reply:
Thanks, split the testing into `test_x86inductor_quantizer.py`.